qi-doc/scribblings/field-guide.scrbl

```diff
@@ -350,6 +354,18 @@ Worse still, even though this computation raises an error, we find that the orig
 
 So in general, use mutable values with caution. Such values can be useful as side effects, for instance to capture some idea of statefulness, perhaps keeping track of the number of times a @tech{flow} was invoked. But they should generally not be used as inputs to a flow, especially if they are to be mutated.
 
+@subsection{Order of Effects}
+
+ Qi flows may exhibit a different order of effects (in the functional programming sense) than equivalent Racket functions.
```

Comment:
…unless those effects are marked with `effect` or `esc`, in which case the order is supposed to be guaranteed, right?